public/style.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css2?family=Jockey+One&display=swap');
```

Comment:
Could we add fonts with `npm` and bundle within our build? Chrome Web Store requires additional reviews if we are importing codes from third-party libraries, even if just for fonts.

Reply:
Hmm, I also tried to add Bootstrap with npm but that didn't work so I ended up using CDN. Can you take a look?